### DISCUSSION TITLE
Is redis-operator compatible with valkey-Images?

### DISCUSSION THREAD

**Discussion by croessner** [OT-CONTAINER-KIT/redis-operator#924] [Q&A]
Hi,

as the license of Redis changed, I would like to use the drop-in replacement valkey. They also provide docker images for valkey and valkey-sentinel. Could I simply replace the images in the values.yaml file and the version tag to use valkey instead of Redis?

Thanks in advance

Christian

**Comment by drivebyer** [comment]
As far as I know, we do not test operators with valkey.

**Comment by arpan57** [comment]
hi @croessner, 

Any luck running valkey on k8s with Redis operator? 
If it worked, can you please share the changes made? 

Thanks,
Arpan

**Reply by croessner** [reply]
Unfortunately I did not find time yet to give it a try.

**Reply by sandeepkunusoth** [reply]
Hi @arpan57 Did u get to try? please let me know

**Reply by arpan57** [reply]
hi @sandeepkunusoth  - yeah it worked for redis-cluster part (Haven't tried other bits).
